smach_viewer/scripts/smach_viewer.py

```diff
@@ -645,7 +645,9 @@ def __init__(self):
         # Wait till we have received the first structure message with a top container
         with self._update_cond:
             while not self._top_containers:
-                self._update_cond.wait()
+                self._update_cond.wait(5.0)
+                print('Waiting for a structure message containing a '
```

Comment:
Combine both lines, please. Setup your editor to accept a line width of 100 chars or so.
Given wide screens, the default limitation of 80 chars is too restrictive.